carthage/deployment.py

```diff
@@ -127,24 +135,73 @@ class MachineDeployableFinder(DeployableFinder):
 
     name = 'machine'
 
-    async def find(self, stop_at):
+    async def find(self, ainjector):
         from .machine import Machine
-        result = await self.ainjector.filter_instantiate_async(Machine, ['host'], stop_at=stop_at, ready=False)
+        result = await ainjector.filter_instantiate_async(Machine, ['host'], stop_at=ainjector, ready=False)
```

Comment:
There's a bit of magic on this `filter_instantiate_async` I need to get my head around still. 

sometimes the 2nd object (predicate) can be a list and sometimes a callable. 

I'm gonna go read the `find` method a bit more to better understand it..